docs/configuration-reference.md

```diff
@@ -10,7 +10,15 @@ The Accelerated Bridge CNI configures networks through a CNI spec configuration
 * `deviceID` (string, required): A valid pci address of a SWITCHDEV NIC's VF. e.g. "0000:03:02.3".
 * `vlan` (int, optional): VLAN ID to assign for the VF. Value must be in the range 0-4094 (0 for disabled, 1-4094 for valid VLAN IDs).
 * `mac` (string, optional): MAC address to assign for the VF
+* `trunk` (array, optional): VLAN trunk configuration for the VF. 
+  Value must be an array of objects with trunk config, e.g.
+  `[{"id": 42}, {"minID": 100, "maxID": 105}, {"minID": 200, "maxID": 210]`,
+  which means that trunk will allow folowing VLANs 42,100-105,200-210
 
+Default VLAN (1) will be used for VF if `vlan` and `trunk` options are not configured.
+It is possible to use `vlan` and `trunk` options together. 
+In this case, VLAN ID from `vlan` option will be used as
+"native" VLAN for VF.
```

Comment:
I don't understand this comment can you elaborate? 
if vlan and trunk are not configure their will be no vlan in the header right?

Reply:
I updated this section in doc.
> if vlan and trunk are not configure their will be no vlan in the header right?
Yes